### DISCUSSION TITLE
Salinity budget in POP2

### DISCUSSION THREAD

**Discussion by Zhangqs-hub** [NCAR/pop-tools#136] [Q&A]
Hi Sir/Lady:
I want to diagnose the salinity budget for the POP2 output, I wonder if you can provide the relevant function or code? Thanks.
Any replies are highly appreciated!
Qiushi

**Comment by dcherian** [comment]
You might have more luck at CESM forums: https://bb.cgd.ucar.edu/cesm/forums/pop.136/
